src/txrequest.h

```diff
@@ -215,6 +215,12 @@ class TxRequestTracker {
 
     /** Access to the internal PriorityComputer (for testing) */
     const PriorityComputer& GetPriorityComputer() const;
+
+    /** Run internal consistency check (test only). */
+    void SanityCheck() const;
+
+    /** Run a time-dependent consistency check (can only be called immediately after GetRequestable; test only). */
+    void TimeSanityCheck(std::chrono::microseconds now) const;
```

Comment:
Renamed to PostGetRequestableSanityCheck, and updated comment.

It's not just a consistency check of request times.

Reply:
Not sure I follow, the only asserts are comparisons between `entry.m_time` and `now` ? Or should I have said "entry times" because "request" times might only mean REQUESTED entries?

Reply:
m_time is the reqtime for CANDIDATE_*, but expiry for REQUESTED entries.